### ISSUE TITLE
BAI Flights RTB without attacking

### ISSUE THREAD

**Issue by holyorangejuice** [dcs-liberation/dcs_liberation#1742]
Before filing, please search the issue tracker to see if the issue has already been reported.

If reporting a DCS AI bug, check https://github.com/dcs-liberation/dcs_liberation#dcs-bugs.

**Describe the bug**
In my current 5.0 campaign I've been able to reproduce BAI flights not attacking targets and instead just RTB.

**To Reproduce**
1. Load my save file
2. Launch a mission

I've been able to replicate it twice in a row. I don't have the time to keep repeating testing. I've got a save file and two tacview files.

**Expected behavior**
BAI should attack first then RTB as usual.

**Additional information**

We will usually need more information for debugging. Include as much of the following as you are able:

https://www.dropbox.com/sh/qy02jbas13aw875/AABskpKArUNrtJGpVxfYky6Aa?dl=0

**Version information (please complete the following information):**
 - DCS Liberation 5.0.0 stable release

Starfire claims he can't replicate this. Which is fine. Though the question remains how do I have proof, two tacview files of it happening, if it can't be replicated. There is an issue somewhere. It needs to be addressed.

**Comment by DanAlbert** [comment]
The save should be enough, thanks.

The usual answer here (I haven't looked) is that the mission has all the right tasking but the DCS AI couldn't detect the target. It happens, and afaik there's nothing we can do about it.

**Comment by Starfire13** [comment]
I have noticed an increased tendency for the AI to RTB without deploying their ordnance when carrying CBUs (this appears to occur whether you use them for BAI or DEAD), though sample count is small so it's not definitive. Not a Liberation issue, even if that's the case. Definitely have not had any issues with them using Mavericks or GBUs for BAI. I even tested using HolyOrangeJuice's Liberation save and his BAI flights (which were carrying GBUs and Mavericks) did their thing flawlessly.

**Comment by holyorangejuice** [comment]
Yeah I'm not sure what it was about that specific situation and I was hoping it would help narrow it down. I've yet to have an issue in other campaigns. I quit that campaign after that issue.
